src/Frontend/Page/Admin.hs

```diff
@@ -0,0 +1,321 @@
+    formPage v fa p = adminFrame p $ do
+        semanticDiv p $ do
+            DF.form v fa $ do
+                div_ $ do
+                    "Wie viele Tage soll die Ausarbaitungphase dauern?" >> br_ []
+                    DF.inputText "elab-duration" v >> "Tage" >> br_ []
+                div_ $ do
+                    "Wie viele Tage soll die Abstimmungphase dauren?" >> br_ []
+                    DF.inputText "vote-duration" v >> "Tage" >> br_ []
+                DF.inputSubmit "AENDERUNGEN SPIECHERN"
+
+adminDurations :: (ActionM m) => ServerT (FormHandler PageAdminSettingsDurations ST) m
+adminDurations = redirectFormHandler (PageAdminSettingsDurations <$> durations) saveDurations
+  where
+    saveDurations :: ActionM m => Durations -> m ()
+    saveDurations (Durations elab vote) = persistent $ do
+        modifyDb dbElabDuration (const elab)
+        modifyDb dbVoteDuration (const vote)
+
+    durations :: ActionM m => m Durations
+    durations = persistent $
+        Durations <$> getDb dbElabDuration
+                  <*> getDb dbVoteDuration
+
+-- * Qourom
+
+instance FormPageView PageAdminSettingsQuorum where
+    type FormPageResult PageAdminSettingsQuorum = Quorums
+
+    formAction _ = relPath $ U.Admin U.AdminQuorum
+
+    redirectOf _ = relPath $ U.Admin U.AdminQuorum
+
+    makeForm (PageAdminSettingsQuorum q) =
+        Quorums
+        <$> ("school-quorum" .: readPercentage defaultSchoolQuorum (schoolQuorumPercentage q))
+        <*> ("class-quorum"  .: readPercentage defaultClassQuorum (classQuorumPercentage q))
+      where
+        readPercentage d v = fromMaybe d . readMaybe <$> DF.string (Just (show v))
+
+    formPage v fa p = adminFrame p $ do
+        semanticDiv p $ do
+            DF.form v fa $ do
+                div_ $ do
+                    "Wie hoch soll das Quorum schulweit sein?" >> br_ []
+                    DF.inputText "school-quorum" v >> "% aller Schulerinnen der Schule" >> br_ []
+                div_ $ do
+                    "Wie hoch soll das Quorum klassenweit sein?" >> br_ []
+                    DF.inputText "class-quorum" v >> "% aller Schulerinnen der Klasse" >> br_ []
+                DF.inputSubmit "AENDERUNGEN SPIECHERN"
+
+adminQourum :: ActionM m => ServerT (FormHandler PageAdminSettingsQuorum ST) m
+adminQourum = redirectFormHandler (PageAdminSettingsQuorum <$> quorum) saveQuorum
+  where
+    saveQuorum (Quorums school clss) = persistent $ do
+        modifyDb dbSchoolQuorum (const school)
+        modifyDb dbClassQuorum (const clss)
+
+    quorum = persistent $
+        Quorums <$> getDb dbSchoolQuorum
+                <*> getDb dbClassQuorum
+
+-- * Groups and permisisons
+
+instance ToHtml PageAdminSettingsGroupsAndPermissions where
+    toHtml = toHtmlRaw
+
+    toHtmlRaw p@(PageAdminSettingsGPUsers users) =
+        adminFrame p . semanticDiv p $ do
+            table_ $ do
+                thead_ . tr_ $ do
+                    th_ "Avarar"
```

Comment:
> I see, so that's why the lower case. But I also meant, this is a typo, "avarar" instead of "avatar".

fixed everything, pr coming up.  thanks @mikolaj!  (: